portfolio/src/main/java/com/google/sps/servlets/CommentForumServlet.java

```diff
@@ -132,7 +133,8 @@ private int getMaxNumComments(HttpServletRequest request) {
   public void storeComment(Comment comment) {
     Entity commentEntity = new Entity("Comment");
     commentEntity.setProperty("comment", comment.getComment());
-    commentEntity.setProperty("timestamp_ms", comment.getTimestamp_ms());
+    commentEntity.setProperty("user-name", comment.getUserName());
```

Comment:
nit: Be consistent with property naming here too. For the timestamp you're using snake_cake but for user-name and sentiment-score you're using kebab-case (I just like these case names). I would actually recommend snake_case just because it's more commonly used.